pkg/rebuild/rebuild/registry.go

```diff
@@ -67,6 +76,13 @@ func warmCacheforArtifact(ctx context.Context, registry RegistryMux, t Target) {
 		registry.CratesIO.Crate(ctx, t.Package)
 		registry.CratesIO.Version(ctx, t.Package, t.Version)
 		registry.CratesIO.Artifact(ctx, t.Package, t.Version)
+	case Debian:
+		component, name, found := strings.Cut(t.Package, "/")
+		if !found {
+			log.Printf("warming cache failed, expected component in debian package name %s", t.Package)
+		}
```

Comment:
the warmups should be in an else or, alternatively, you could early return in the !found